docs/source/migration/3.0.mdx

```diff
@@ -1,50 +1,57 @@
 ---
-title: Migrating to 3.0
-description: Migrate to Apollo Android 3.0
+title: Migrating to Apollo Android 3.0
+description: From 2.x
+sidebar_title: "Migrating to v3"
 ---
 
 import {MultiCodeBlock} from 'gatsby-theme-apollo-docs';
 
-Apollo Android 3.0 is a rewrite of most of the internals of Apollo Android in Kotlin with, amongst other things:
+Apollo Android 3.0 rewrites most of Apollo Android's internals in Kotlin. Among other improvements, it features:
 
-* Kotlin-first, coroutine-based APIs.
-* A unified runtime for both JVM and multiplatform.
+* Kotlin-first, coroutine-based APIs
+* A unified runtime for both JVM and multiplatform
 
-While the concepts stay the same, a lot of APIs have changed to work better in Kotlin. This page presents the important changes and how to migrate from Apollo Android 2.x to Apollo Android 3.x
+Although most of the library's concepts are the same, many APIs have changed to work better in Kotlin.
 
-## `com.apollographql.apollo3` is the new package name, group id and plugin id
+This page describes the most important changes, along with how to migrate an existing project from Apollo Android 2.x to Apollo Android 3.x.
 
-In order to avoid conflicting classes in the classpath and dependencies conflicts, and according to the [interoperability policy for major version updates](https://jakewharton.com/java-interoperability-policy-for-major-version-updates/), Apollo Android 3.0 uses a new maven group id and package name: `com.apollographql.apollo3`
+## Package name / group id / plugin id
 
-**Group id:**
+Apollo Android 3.0 uses a new identifier for its package name, maven group id, and Gradle plugin id: **`com.apollographql.apollo3`**.
 
-The maven group id used to identify the [artifacts](https://repo1.maven.org/maven2/com/apollographql/apollo3/) is now `com.apollographql.apollo3`
+This change helps avoid dependency conflicts and conflicting classes in the classpath, as encouraged in [Java Interoperability Policy for Major Version Updates](https://jakewharton.com/java-interoperability-policy-for-major-version-updates/).
+
+> In most cases, you can update identifier throughout your project by performing a find-and-replace on the 2.x identifier, `com.apollographql.apollo`.
+
+#### Package name
```

Comment:
I would change the order there to have:

1. Group ID
2. Plugin ID
3. Package name

While the package name is the most important change, the order in which the replacements will be done is going to be the one above. i.e. you first need to find the correct artifact then fix the plugin for the IDE to sync. Only then you can work in the code and start changing the package names